### DISCUSSION TITLE
Plots end date

### DISCUSSION THREAD

**Discussion by kdrobnyh** [babybuddy/babybuddy#353] [General]
Hey!

Some plots (e.g., Feeding Durations and Diaper Change Types) do not end on the current date. E.g., 1 week plot ends on Dec 16 and starts on Dec 10 (now it's Dec 13). Is that the expected behavior or a bug?

**Comment by cdubz** [comment]
What’s the oldest entry date? I can’t remember off the top of my head but that may be the default behavior when there is not a full week of data available?

**Reply by kdrobnyh** [reply]
There is 2+ months data available, so I do not think that's the issue. From my phone, it's from Dec 12 to Dec 18. 
If you think that sounds like a bug, I can investigate it.

**Reply by cdubz** [reply]
Weird. Yeah I would say that is a bug if the behavior differs on phone vs. desktop. Though I wonder if the issue is with Plotly itself...

**Reply by kdrobnyh** [reply]
Looks like it's an issue with Plotly. If it's a scatterplot, the x axis is extended for some extra days before and after the range of the given dataset. I assumed that by clicking on '1w' only the last 7 available days should be shown, but apparently it's not the case.

**Reply by cdubz** [reply]
Yeah that Plotly config has always been a little awkward. 🤷
